h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostUtils.java

```diff
@@ -870,7 +869,7 @@ private static SparseMatrixDimensions calculateCSRMatrixDimensions(Frame f, int[
             }
         }
 
-        return new SparseMatrixDimensions(nonZeroElementsCount, ++rowIndicesCount);
+        return new SparseMatrixDimensions(nonZeroElementsCount, rowIndicesCount);
```

Comment:
Regarding changing from ++rowIndicesCount to rowIndicesCount, I found there are cases where the array size is off by one.  Hence, I switch this one here.